src/main/java/helper/DriverHelper.java

```diff
@@ -23,54 +23,48 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.Select;
 import org.openqa.selenium.support.ui.WebDriverWait;
-//import org.openqa.selenium.Alert;
 
 public class DriverHelper implements WebDriver {
 
 	public static WebDriver driver;
 	private final static WebDriverWait wait;
-    public static final String browser = System.getProperty("browser");
-    public static final String js = System.getProperty("js");
-    
+	public static final String browser = System.getProperty("browser");
+	public static final String js = System.getProperty("js");
+
 	public DriverHelper() {
 
 	}
 
 	static {
 		if (browser.equalsIgnoreCase("firefox")) {
-			if(js.equalsIgnoreCase("on")){
+			if (js.equalsIgnoreCase("on")) {
 				driver = new FirefoxDriver();
-			}else if(js.equalsIgnoreCase("off")){
+			} else if (js.equalsIgnoreCase("off")) {
 				FirefoxProfile profile = new FirefoxProfile();
-		    	profile.setPreference("javascript.enabled", false);
-		    	driver = new FirefoxDriver(profile);
-			}else{
+				profile.setPreference("javascript.enabled", false);
+				driver = new FirefoxDriver(profile);
+			} else {
```

Comment:
recheck why there is a biiiiig space between profile and setPreference and so on